packages/algob/src/lib/msig.ts

```diff
@@ -1,5 +1,6 @@
 import { getPathFromDirRecursive } from "@algo-builder/runtime";
-import type { Account, LogicSig, MultiSig, TxSig } from "algosdk";
+import { types as wtypes } from "@algo-builder/web";
+import type { Account, EncodedMultisig, LogicSig } from "algosdk";
```

Comment:
linter warning: `'wtypes' is defined but never used. Allowed unused vars must match /_/u`

Reply:
removed